frontend/src/pages/login/index.tsx

```diff
@@ -6,7 +6,7 @@ const Login = () => {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
 
-  const handleChangeEmail = (event: React.FormEvent<HTMLInputElement>) => {
+  const emailChangeHandler = (event: React.FormEvent<HTMLInputElement>) => {
```

Comment:
Previous name seemed more convenient

Reply:
Handler function names are usually: `handle...change` or in this case `handleEmailChange`